src/core_modules/capture-core/components/Pages/Search/SearchDomainSelector/SearchDomainSelector.component.js

```diff
@@ -7,82 +7,64 @@ import {
     SingleSelect,
     SingleSelectOption,
 } from '@dhis2/ui';
-import { Section, SectionHeaderSimple } from '../../../Section';
 import type { OwnProps, Props } from './SearchDomainSelector.types';
+import { searchScopes } from '../SearchPage.constants';
 
-const styles = (theme: Theme) => ({
+const styles = ({ typography }) => ({
     searchDomainSelectorSection: {
-        margin: theme.typography.pxToRem(10),
+        margin: typography.pxToRem(10),
+    },
+    header: {
+        paddingLeft: 8,
     },
     searchRow: {
+        maxWidth: typography.pxToRem(400),
         display: 'flex',
         alignItems: 'center',
         justifyContent: 'center',
-    },
-    searchRowTitle: {
-        flexBasis: 200,
-        marginLeft: 8,
+        padding: '8px 0',
     },
     searchRowSelectElement: {
+        marginLeft: 8,
+        marginRight: 8,
+        marginBottom: 8,
         width: '100%',
     },
     customEmpty: {
         textAlign: 'center',
         padding: '8px 24px',
     },
-    divider: {
-        padding: '8px',
-    },
 });
 
 export const Index =
   ({ trackedEntityTypesWithCorrelatedPrograms, classes, onSelect, selectedSearchScopeId }: Props) =>
-      (<Section
-          className={classes.searchDomainSelectorSection}
-          header={
-              <SectionHeaderSimple
-                  containerStyle={{ paddingLeft: 8, borderBottom: '1px solid #ECEFF1' }}
-                  title={i18n.t('Search scope')}
-              />
-          }
-      >
-          <div className={classes.searchRow} style={{ padding: '8px 0' }}>
-              <div className={classes.searchRowTitle}>{ i18n.t('Find results from') }</div>
-              <div className={classes.searchRowSelectElement} style={{ marginRight: 8 }}>
+      (<>
```

Comment:
Basically in this component I've updated the component to match the design from the concept.